packages/react-components/src/components/CameraButton.tsx

```diff
@@ -147,6 +147,11 @@ export interface CameraButtonProps extends ControlBarButtonProps {
    * Styles for {@link CameraButton} and the device selection flyout.
    */
   styles?: Partial<CameraButtonStyles>;
+  /* @conditional-compile-remove(video-background-effects) */
+  /**
+   * Callback when a effects is clicked
+   */
+  onClickVideoEffectsButton?: (showPanel: boolean) => void;
```

Comment:
Maybe we don't need the panel state as a callback arg. The CameraButton component should not know anything about the panel. But interested in what others think.

Reply:
This callback can be used to show/hide video effects options. I have changed it to `showVideoEffectsOptions`